app/questionnaire/questionnaire_manager.py

```diff
@@ -30,13 +30,11 @@ def __init__(self, schema, json=None):
         self._schema = schema
         self.state = None
 
-        self.navigator = Navigator(self._json)
```

Comment:
Can we still set a navigator property on questionnaire manager, but pass it in instead of initialising it on creation. This would make the navigator usage consistent with the global json and schema.